cmd/cloud/deployment.go

```diff
@@ -360,3 +371,50 @@ func deploymentVariableUpdate(cmd *cobra.Command, args []string, out io.Writer)
 
 	return deployment.VariableModify(deploymentID, variableKey, variableValue, ws, envFile, variableList, useEnvFile, makeSecret, true, astroClient, out)
 }
+
+func newDeploymentWorkerQueueRootCmd(out io.Writer) *cobra.Command {
```

Comment:
worker queue commands can be part of `deployment_workerqueue.go` so that this file doesn't end up being very long once other worker queue CRUD commands are being added

Reply:
like this and will refactor workerqueue CRUD commands to their own file.